src/General/Badge/Badge.tsx

```diff
@@ -13,6 +13,7 @@ const Badge: React.FunctionComponent<Props> = ({
     {!isNil(label) && (
       <BadgeContainer
         className={classNames('aries-badge', className)}
+        role="presentation"
```

Comment:
why name the role `presentation`?

Reply:
Because it had the same name in the rest of the components, so followed that :stuck_out_tongue: